packages/wallet-service/src/api/healthcheck.ts

```diff
@@ -0,0 +1,145 @@
+import middy from '@middy/core';
+import {
+    Healthcheck,
+    HealthcheckInternalComponent,
+    HealthcheckDatastoreComponent,
+    HealthcheckHTTPComponent,
+    HealthcheckCallbackResponse,
+    HealthcheckStatus,
+} from '@hathor/healthcheck-lib';
+import { getLatestHeight } from '@src/db';
+import fullnode from '@src/fullnode';
+import { closeDbConnection, getDbConnection } from '@src/utils';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { getRedisClient, ping } from '@src/redis';
+
+const mysql = getDbConnection();
+
+const HEALTHCHECK_MAXIMUM_HEIGHT_DIFFERENCE = Number(process.env.HEALTHCHECK_MAXIMUM_HEIGHT_DIFFERENCE) || 5;
```

Comment:
Is there any chance that we want to use `process.env.HEALTHCHECK_MAXIMUM_HEIGHT_DIFFERENCE = '0'` someday?

In this case this would be `5`, not `0`.

Reply:
You're right. I think we should allow setting as 0, this is a bug. I'll fix this.

Reply:
Fixed in https://github.com/HathorNetwork/hathor-wallet-service-sync_daemon/pull/110/commits/e078a4f4f7e3ff8114b37e9bfce8cad6ccac78d5